website/content/en/docs/advanced-topics/multi-arch.md

```diff
@@ -4,12 +4,17 @@ linkTitle: Multiple Architectures
 weight: 80
 ---
 
-### Supporting Multiple Architectures
+# Supporting Multiple Architectures
 
 Authors may decide to distribute their bundles for various architectures: x86_64, aarch64, ppc64le, s390x, etc, to accommodate the diversity of Kubernetes clusters and reach a larger number of potential users. Each architecture requires however compatible binaries.
 
-#### Manifest lists
+## Fundamentals
+The basics of supporting multiple platforms is to ensure that each of your operator images are built for each of the platforms to be supported. From there, the images should be hosted in image registries as manifest lists. Finally, you'll made need to update your distribution configuration to set which platforms are supported. This section explains each of these concepts in turn.
```

Comment:
is "platform" the correct term here? maybe "architecture platforms"? or just "architectures" ?

nit: `Finally, you'll need `